### PR TITLE
Monitoring: Update silence form text and styles

### PR DESCRIPTION
**Pull request by kyoto**
Updates the Create Silence and Edit Silence pages with various text,
layout and styling changes.

FYI @cshinn 

screenshot

### CODE REVIEW

File: frontend/public/components/_monitoring.scss [hunk 1]
```diff
@@ -30,6 +30,10 @@ $monitoring-line-height: 18px;
   text-overflow: ellipsis;
 }
 
+.monitoring-no-padding-left {
```
Comment (rhamilto) [inline frontend/public/components/_monitoring.scss:33]: This is not a great name.  Maybe `monitoring-reg-ex` or something that speaks to the functionality?

File: frontend/public/components/_monitoring.scss [hunk 2]
```diff
@@ -44,6 +48,11 @@ $monitoring-line-height: 18px;
   padding: 0;
 }
 
+.btn--silence-remove {
```
Comment (rhamilto) [inline frontend/public/components/_monitoring.scss:51]: Also not a great name as `silence-remove` is more of an element name, not a modifier name.  I'd use just a className that describes the button itself.  I realize you're following an existing convention in this file (line 47), but it seems like a good time to clean up both instances.

File: frontend/public/components/monitoring.tsx [hunk 3]
```diff
@@ -1273,55 +1273,75 @@ class SilenceForm_ extends React.Component<SilenceFormProps, SilenceFormState> {
...
+                      />
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div className="co-m-pane__body-group">
+              <div className="form-group row">
+                <div className="col-xs-9">
```
Comment (rhamilto) [inline frontend/public/components/monitoring.tsx:1321]: Any reason to not allow this to grow to the full 12 cols (`col-xs-12`) in order to take up less vertical space?

Comment (kyoto) [reply frontend/public/components/monitoring.tsx:1321]: Just following the new mocks for this page.

I don't have a preference, but if @cshinn feels it's better to make this text full width, I can change it (probably a follow-up PR?).

File: frontend/public/components/_monitoring.scss (hunk 2 above)
Comment (kyoto) [reply frontend/public/components/_monitoring.scss]: Changed to `monitoring-silence-remove-button`.

I also realized that `btn--silence-add-more` is no longer used, so removed that.

File: frontend/public/components/_monitoring.scss (hunk 1 above)
Comment (kyoto) [reply frontend/public/components/_monitoring.scss]: Changed to `monitoring-silence-reg-ex`

**Comment by kyoto** [comment]
Thanks @rhamilto! I addressed your comments.

**Pull request merged**